projects/plugins/jetpack/_inc/lib/class-jetpack-recommendations.php

```diff
@@ -217,6 +222,35 @@ public static function plugin_activated( $plugin ) {
 		}
 	}
 
+	/**
+	 * Runs when a new comment is added.
+	 *
+	 * @param integer $comment_id The ID of the comment that was added.
+	 * @param bool    $comment_approved Whether or not the comment is approved.
+	 * @param array   $commentdata Comment data.
+	 */
+	public static function comment_added( $comment_id, $comment_approved, $commentdata ) {
+		if ( ! function_exists( 'is_plugin_active' ) ) {
+			require_once ABSPATH . '/wp-admin/includes/plugin.php';
+		}
+
+		if ( is_plugin_active( 'akismet/akismet.php' ) || self::is_conditional_recommendation_enabled( self::ANTI_SPAM_RECOMMENDATION ) ) {
```

Comment:
Maybe we could rely on `Plugins_Installer::is_plugin_active()` here?

Reply:
Ah, good idea :)